great-idea-website/css/index.css

```diff
@@ -66,13 +66,19 @@ h1, h2, h3, h4, h5 {
 /* Your code starts here! */
 
 .main-container {
-	margin: 50px;
+	max-width: 1140px;
+	margin-left: auto;
+	margin-right: auto;
 }
 
```

Comment:
Great job on your fixed nav. One way you can push yourself on this is thinking about how to make the bar go all the way across the window even if the window is wider than your container div. Here's a hint: just because it's called the main-container, doesn't mean everything has to go in it.

Reply:
@theryanclausen, I will look into making that improvement. Thank you!

Reply:
@theryanclausen, updated nav bar. Thanks for the suggestion! Looks much better.